docs/_templates/breadcrumbs.html

```diff
@@ -1,4 +0,0 @@
-{%- extends "sphinx_rtd_theme/breadcrumbs.html" %}
-
-{% block breadcrumbs_aside %}
-{% endblock %}
```

Comment:
We had a custom breadcrumbs.html to remove some feature, probably an "edit on GitHub" link that didn't work because of how things were set up. Is all of that obsolete now?

Reply:
This currently doesn't do anything as we don't use `sphinx_rtd_theme`. However, `pydata_sphinx_theme` just added breadcrumbs, so now this no-op template loads and breaks things. We should probably re-introduce the feature that this template provided, on a more selective basis; generated listings, unlike the user guide, are not editable. I'll look into that in a new PR :)